app/forms/base_add_another_form.rb

```diff
@@ -15,7 +15,7 @@ class BaseAddAnotherForm
 
   class << self
     def from_session(session_data)
-      form = new
+      form = new.tap { _1.session_data = session_data }
```

Comment:
We don't, I'm being an idiot.  I thought because session_data was a ruby `attr_accessor` rather than an ActiveModel `attribute` that we couldn't pass it to `new` as an argument like we can for other attributes, and I was just wrong. Fixing now.